lib/src/Base/Stat/openturns/CovarianceModelImplementation.hxx

```diff
@@ -199,6 +197,9 @@ public:
   /** Method load() reloads the object from the StorageManager */
   void load(Advocate & adv) override;
 
+  /** outputCovariance_ accessor*/
+  CovarianceMatrix getOutputCovariance() const;
```

Comment:
this new method is not documented

Reply:
Ah, sorry, I added this method to debug the changes and forgot to remove it. I will do it shortly.